adapters/eplanning/eplanning.go

```diff
@@ -61,16 +65,62 @@ type hbResponseAd struct {
 	Height       uint64 `json:"h,omitempty"`
 }
 
+func isPriority(index1 int, index2 int) bool {
+	if index1 > -1 {
+		if index2 > -1 {
+			if index1 < index2 {
```

Comment:
This is essentially the return for `less`. Why is it returning false when index1 is less than index2? Could you please explain the algorithm and/or perhaps add comments?